packages/react/package.json

```diff
@@ -119,6 +127,41 @@
       "@commitlint/config-conventional"
     ]
   },
+  "nyc": {
+    "report-dir": "cypress/coverage",
+    "reporter": [
+      "html",
+      "text-summary",
+      "lcov",
+      "json"
+    ],
+    "include": [
+      "src/components/**/*.js?(x)"
+    ],
+    "exclude": [
+      "src/**/*.story.js?(x)",
+      "src/**/hooks/*.js",
+      "src/components/SuiteHeader/util/suiteHeaderData.js",
+      "src/components/SuiteHeader/util/uiresources.js",
+      "src/components/FileUploader/stories/*.jsx",
+      "src/components/Table/AsyncTable/*.js?(x)",
+      "src/components/WizardInline/**/*.js?(x)",
+      "src/components/Page/(EditPage|PageHero).jsx",
+      "src/components/Dashboard/(Dashboard|CardRenderer).jsx",
+      "src/components/MapCard/storyFiles/*.jsx",
+      "src/components/BarChartCard/barChartUtils.js",
```

Comment:
This may need to be revisited. I had originally started with just copying the items from Jest config but I was noticing threshold failures for these other files. I think as part of our effort to bring the codebase completely up to 90% we can look at these exceptions and either verify that they should be ignored or write test to bring the threshold up properly